pkg/config/config.go

```diff
@@ -55,8 +55,9 @@ var (
 // Config holds process global configuration information
 type Config struct {
 	// System config
-	EtcdSecretName    string // DEPRECATED - should be removed in the future
-	ModelMeshEndpoint string // For dev use only
+	EtcdSecretName              string // DEPRECATED - should be removed in the future
+	ModelMeshEndpoint           string // For dev use only
+	EnableDeployPVCForPredictor bool
```

Comment:
I think we should name this differently, how about `AllowAnyPVC`?

We should also add to the docs ... the config params table and also in the storage part of the docs explain how this parameter works.